fizzbuzz.js

```diff
@@ -1,12 +1,21 @@
+const Error = require('./statusCodes.js');
+
 const fizzbuzz = (number) => {
-  if ((number % 3 === 0) && (number % 5 === 0)) {
+  if (typeof number === 'undefined'){
+    return (Error.NullValue);
+  } else if (number > 100){
+    return (Error.ErrorString);
+  } else if (typeof number === 'string'){
+    return (Error.ErrorLargeNumber);
+  } else if ((number % 3 === 0) && (number % 5 === 0)) {
       return 'fizzbuzz';
   } else if (number % 3 === 0) {
       return 'fizz';
   } else if (number % 5 === 0) {
       return 'buzz';
+  } else {
+      return number;
   }
-  return number;
 }
 module.exports = {
```

Comment:
you can export any data type you want on module.exports (think of it this way: fundamentally, `module` is an object, with an `exports` property. Properties can be anything - null, strings, functions, objects, etc. 

Here, since you're only exporting the fizzbuzz function, you should export the function directly, vs. embedding it in an object.